app/src/androidTest/AndroidManifest.xml

```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
-<manifest>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+   >
 
+    <!-- For test coverage -->
+    <uses-permission
+        android:name="android.permission.WRITE_EXTERNAL_STORAGE"
+        android:maxSdkVersion="29" />
+
+    <!-- For test coverage on API 29 only -->
+    <application
+        android:requestLegacyExternalStorage="true" />
```

Comment:
I wonder if we need to change app/manifest if we don't run test with coverage against `app` module? The question also applies to the `ui-screenshot-test` module. Also, it applies to other changes in the app Gradle files of these two modules.

Reply:
We should be able to generate a coverage report for the app module when it is being run under instrumentation tests.  We just can't run the debug app apk with coverage injected because it'll crash on startup.